services/workflows-service/src/business/business.module.ts

```diff
@@ -54,6 +56,8 @@ import { WorkflowTokenRepository } from '@/auth/workflow-token/workflow-token.re
     SalesforceIntegrationRepository,
     WorkflowTokenService,
     WorkflowTokenRepository,
+    UiDefinitionRepository,
+    UiDefinitionService,
```

Comment:
Could use help with using the `UIModule` instead. Kept getting different errors.